src/annotator/adder.js

```diff
@@ -130,7 +138,7 @@ export class Adder {
    * Return the best position to show the adder in order to target the
    * selected text in `targetRect`.
    *
-   * @param {Rect} targetRect - The rect of text to target, in viewport
+   * @param {DOMRect} targetRect - The rect of text to target, in viewport
```

Comment:
The `Rect` type wasn't defined anywhere. [DOMRect](https://developer.mozilla.org/en-US/docs/Web/API/DOMRect) the name of the type returned by various DOM APIs that return rects.